src/core/main_task.c

```diff
@@ -29,6 +36,9 @@ uint8_t extra_param;
 static void process_alarm(uint32_t notification);
 static void process_critical_failure(uint32_t notification);
 
+#define WRITE_EEPROM_PERIOD_MS 3000 * 1000L // Try 5 * 1000L for Testing purpose
+#define EEPROM_INIT_MAGIC_NUMBER 1
```

Comment:
Use a random number as magi number (othewise there is a risk of finding it correct by chance).
```suggestion
#define EEPROM_INIT_MAGIC_NUMBER 0xc417b8c6
```